__tests__/app.test.js

```diff
@@ -210,4 +211,56 @@ describe("PATCH /api/articles/:article_id", () => {
             expect(body.msg).toBe("article not found!")
         })
     })
+})
+describe("/api/articles(queries)", () => {
+    test("GET 200 - filters the articles by specified topic", () => {
+        return request(app).get("/api/articles?topic=mitch").expect(200).then(({body}) => {
+            expect(body.articles).toHaveLength(11)
+            body.articles.forEach((article) => {
+                expect(article.topic).toBe("mitch")
+            })
+        })
```

Comment:
Just missing one test for a valid topic query that has no associated articles. What should be returned for a topic that exists in the database but has no articles? Currently, your code defaults to an error if the returned array is empty, but this isn't a bad request if the topic query exists in the database.